kubeproxy/create-cluster.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
```

Comment:
This seems like a really useful script that can be used to establish e2e/smoketests for all operators.
Should this be moved outside to a `test/` dir? 
https://github.com/kubernetes-sigs/cluster-addons/pull/63 might give some more context

Reply:
The script is for creating a kinder cluster without kubeproxy. I could make a PR for one that creates a normal kind or kinder cluster with kube-proxy already installed. Wdyt?

Reply:
>The script is for creating a kinder cluster without kubeproxy. 

Yes, this is exactly what's desired. My thinking is once this script creates the cluster, the existing `smoketest.go` can run the tests on it.

Reply:
> > The script is for creating a kinder cluster without kubeproxy.
> 
> Yes, this is exactly what's desired. My thinking is once this script creates the cluster, the existing `smoketest.go` can run the tests on it.

Oooh, I see what you mean. @johnsonj wdyt?

Reply:
Totally makes sense to me. We'll always need our own `kube-up.sh`, even in a world where these operators are default we need a way to test with our versions!

Now the question I'd pose is: Copy the shell script, or reimplement it in Go?

Reply:
I think that the shell script should suffice